Plots/Station/NCL_station_3.py

```diff
@@ -0,0 +1,157 @@
+"""
+NCL_station_3.py
+================
+This script illustrates the following concepts:
+   - Drawing station numbers on a map, and removing ones that overlap
+   - Attaching lots of text strings to a map
+
+See following URLs to see the reproduced NCL plot & script:
+    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/station_3.ncl
+    - Original NCL plot: https://www.ncl.ucar.edu/Applications/Images/station_3_1_lg.png and https://www.ncl.ucar.edu/Applications/Images/station_3_2_lg.png
+"""
+
+###################################################
+# Import packages:
+
+import numpy as np
+import pandas as pd
+import cartopy.crs as ccrs
+import cartopy.feature as cfeature
+from matplotlib import pyplot as plt
+import matplotlib.ticker as mticker
+
+import geocat.datafiles as gdf
+
+###################################################
+# Generate data:
+
+# Open a netCDF data file using xarray default engine and load the data into xarrays
```

Comment:
I think you should consider reading this in with xarray instead of pandas, like we do in newer examples.

Reply:
I don't think xarray's open_dataset is meant for txt files. I had erros opening txt files with it:
 _did not find a match in any of xarray's currently installed IO backends ['netcdf4', 'scipy']. Consider explicitly selecting one of the installed backends via the ``engine`` parameter to xarray.open_dataset(), or installing additional IO dependencies:_

Another function to use for opening txt files are numpy's load_txt, but I think it is more cumbersome because we have to specify datatypes for each column when both floats and strings exist.

Let me know if I'm wrong about this!

Reply:
You're right I apologize! However, could you change the comment to reflect that you are using pandas instead of xarray

Reply:
Nevermind, I see you've already done this!